omniport/omniport/middleware/routes_control.py

```diff
@@ -42,4 +44,15 @@ def __call__(self, request):
 
         response = self.get_response(request)
 
+
+        for app, app_configuration in DISCOVERY.apps():
+            base_url  = app_configuration.base_urls.http.strip('/')
+            if not re.math(f'^/{base_url}/', request.path):
+                continue
+            if not from_acceptable_ring(
+                    source_address,
+                    app_configuration.acceptables.ip_address_rings
+                ):
+                raise HttpResponseForbidden()
```

Comment:
If I recall correctly this does not derive from `Exception` so it must be `return`-ed instead of `raise`-d.

Reply:
Ok. I'll update the changes